library/os_ironic_state.py

```diff
@@ -71,11 +71,12 @@
 
 from distutils.version import StrictVersion
 
+# Store a list of import errors to report to the user.
+IMPORT_ERRORS = []
```

Comment:
Looks like `IMPORT_ERRORS` is only ever going to have zero or one element, is this pattern in case more imports are added in the future?